dashboard/analysis_options.yaml

```diff
@@ -28,4 +28,5 @@ analyzer:
 include: package:flutter_lints/flutter.yaml
 linter:
   rules:
-    prefer_single_quotes: true
\ No newline at end of file
+    prefer_single_quotes: true
+    # require_trailing_commas: true
```

Comment:
nit: Files should a newline at the end of them

Reply:
Got it. I will remove this.